app/models/spotlight/exhibit.rb

```diff
@@ -56,8 +57,6 @@ class Exhibit < ActiveRecord::Base
     after_create :initialize_main_navigation
     include Spotlight::DefaultThumbnailable
 
-    scope :published, -> { where(published: true) }
```

Comment:
This line is duplicated on line 9. It probably should be in a separate commit, though.